lib/utils.pm

```diff
@@ -803,6 +803,10 @@ sub power_action {
         $shutdown_timeout *= 3;
         record_soft_failure("boo#1057637 shutdown_timeout increased to $shutdown_timeout (s) expecting to complete.");
     }
+    # Override the timeout using a value from DEBUG_SHUTDOWN parameter.
```

Comment:
This comment can be removed. The code is explicit enough.

Reply:
Done.